CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+# We love contributions!
+
+How can I contribute?
+
+There are many ways you can contribute to Kogito Serverless Workflow Operator, not only software development, as well as with the rest of Kogito community:
+
+    Contribute actively to development (see the section below)
+
+    Use it and report any feedback, improvement or bug you may find via Github, mailing list or chat.
+
+    Contribute by writing missing documentation or blog posts about the features around Kogito
+
+    Tweet, like and socialize Kogito in your preferred social network
+
+    Enjoy the talks that the contributors submit in various conferences around the world
+
+
+# Contributing to the Kogito Serverless Operator codebase
+
+The main project is written in go. 
+Kogito Serverless Workflow Operator is built on top of Kubernetes through Custom Resource Definitions.
+
+- Workflow
+- Platform
+- Build
+
+This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+
+It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/)
+which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster
+
+
+# Contributing to the Kogito Serverless Operator
+
+# PreRequisites
+The Operator's controllers and the configurations are generated using the Operator sdk, the code is written in Go, the tasks are executed using a Makefile
+
+In order to build the project, you need to comply with the following requirements:
+
+operator-sdk-v1.24.0 (https://sdk.operatorframework.io/docs/building-operators/golang/installation/)
+
+More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
+
+Go 1.19 (https://go.dev/dl/)
+
+GNU Make: 
+used to define composite build actions. This should be already installed or available as a package if you have a good OS (https://www.gnu.org/software/make/).
+
+**NOTE:** Run `make help` for more information on all potential `make` targets
+
+
+## Getting Started
+You’ll need a Kubernetes cluster to run against. You can use 
+- [KIND](https://sigs.k8s.io/kind) 
+- [MINIKUBE](https://minikube.sigs.k8s.io)  
+- [CRC](https://console.redhat.com/openshift/create/local) 
+
+to get a local cluster for testing, or run against a remote cluster.
+**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
```

Comment:
+1

Even if I don't understand the note. It depends where you will execute the controller:
- if you execute it locally it's true
- if you deploy it in a Pod it will use the context for that Pod

Since we are in the Getting started section I don't think that the note it is really meaningful here.

It could be useful instead in the [Test it out](https://github.com/davidesalerno/kogito-serverless-operator/tree/98defe6f1333627982775ffc4dd3363134114f55#test-it-out) section